shimmingtoolbox/langauge/French.py

```diff
@@ -0,0 +1,43 @@
+_array_3d = 'Les tableaux d\'entrée X, Y et Z doivent être 3d'
```

Comment:
we don't care about French translation. Scientists should know English and this is unnecessary entries for us to maintain. We have _much more_ important things to take care of at this stage of development

Reply:
Was going to pull this out. The point here is to pull messages out of the code to improve readability, and this was just an extra.